src/components/GamesList.jsx

```diff
@@ -19,7 +20,7 @@ const GamesList = () => {
             {reviews.map((review) => {
                 return(
                     <section className="GamesList__GamesCard" key={review.review_id}>
-                        <h3 className="GamesCard__h3">{review.title}</h3>
+                        <Link className="GamesCard__h3" key={review.review_id} to={`/reviews/${review.review_id}`}><h3>{review.title}</h3></Link>
```

Comment:
It's up to you but you could wrap the link around the whole section. That way users can click the card itself to naviagte :)

Reply:
I was considering this but I think it might break something I'm considering adding later